.travis.yml

```diff
@@ -1,9 +1,16 @@
 language: python
 
+env:
+  - TOXENV=quality
+
 matrix:
   include:
-  - python: 2.7
-    env: TOXENV=quality
+    - python: 2.7
+    - python: 3.6
+    - python: 3.7
+      dist: xenial  # required for Python >= 3.7 (travis-ci/travis-ci#9069)
+  allow_failures:
+    - python: 3.7   # RuntimeError: generator raised StopIteration
```

Comment:
I think that's a bug fixed in later versions of astroid or pylint; we can't upgrade those any further without losing support for Python 2.7, so fixing that will have to wait for a while.

Reply:
Conditional versions seems to get around this issue:
* https://github.com/edx/edx-val/pull/170/files#diff-15c4d16f426f922ec58083a922ec42b6R11